packages/snet_cli/snet/snet_cli/mpe_service_metadata.py

```diff
@@ -85,21 +82,80 @@ def set_simple_field(self, f, v):
             raise Exception("unknown field in MPEServiceMetadata")
         self.m[f] = v
 
-    def set_fixed_price_in_cogs(self, price):
+    def set_fixed_price_in_cogs(self, group_name, price):
         if (type(price) != int):
             raise Exception("Price should have int type")
-        self.m["pricing"] = {"price_model": "fixed_price",
-                             "price_in_cogs": price}
 
-    def add_group(self, group_name, payment_address):
+        if (not self.is_group_name_exists(group_name)):
+            raise Exception("the group %s is not present" % str(group_name))
+
+        for group in self.m["groups"]:
+            if group["group_name"] == group_name:
+                # default=True  it will change when we will go live with method level pricing
+                if "pricing" in group:
+                    group["pricing"].append({"price_model": "fixed_price",
+                                             "price_in_cogs": price, "default": True})
+                else:
+                    group["pricing"] = [{"price_model": "fixed_price",
+                                         "price_in_cogs": price, "default": True}]
+
+    def set_method_price_in_cogs(self, group_name, package_name, service_name, method, price):
```

Comment:
Are we including method level pricing as well? Are the complete changes in for this?

Reply:
For creating metadata changes are complete , we will not be picking it up in pricing as of now.